ubuntu-20/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # 'UBUNTU_VERSION' below needs a default value, and is replaced by build-arg in Makefile
```

Comment:
I would have thought that we'd create a new dir for this file (`ubuntu-22/Dockerfile`) as the name is now misleading

Reply:
Well ... its less to edit where the image is used in 80 or so repos ...